grails-app/conf/Config.groovy

```diff
@@ -269,9 +269,22 @@ environments {
  * -Dstreamr.ethereum.networks.someNetwork=http://some-network-rpc-url
  * -Dstreamr.ethereum.networks.anotherNetwork=http://some-network-rpc-url
  */
+// default network restricted in infura.io to following addresses, see https://infura.io/project/117d24d0fef1434fbd14567435a363d9
+// rinkeby "DATAcoin": 0x8e3877Fe5551f9c14BC9b062bbae9D84bC2F5d4E
+// rinkeby Marketplace: 0x0af64558670a3b761B57e465Cb80B62254b39619
+// mainnet DATAcoin: 0x0Cf0Ee63788A0849fE5297F3407f701E122cC023
+// mainnet Marketplace: 0xA10151D088f6f2705a05d6c83719e99E079A61C1
 streamr.ethereum.defaultNetwork = System.getProperty("streamr.ethereum.defaultNetwork") ?: "local"
-streamr.ethereum.networks = PropertiesUtil.matchingPropertiesToMap("streamr.ethereum.networks.", System.getProperties()) ?: [local: "http://localhost:8545"]
-streamr.ethereum.wss = PropertiesUtil.matchingPropertiesToMap("streamr.ethereum.wss.", System.getProperties()) ?: [local: "ws://localhost:8545"]
+streamr.ethereum.networks = PropertiesUtil.matchingPropertiesToMap("streamr.ethereum.networks.", System.getProperties()) ?: [
+	local: "http://localhost:8545",
+	rinkeby: "https://rinkeby.infura.io/v3/117d24d0fef1434fbd14567435a363d9",
+	mainnet: "https://mainnet.infura.io/v3/117d24d0fef1434fbd14567435a363d9",
+]
+streamr.ethereum.wss = PropertiesUtil.matchingPropertiesToMap("streamr.ethereum.wss.", System.getProperties()) ?: [
+	local: "ws://localhost:8545",
+	rinkeby: "wss://rinkeby.infura.io/ws/v3/117d24d0fef1434fbd14567435a363d9",
+	mainnet: "wss://mainnet.infura.io/ws/v3/117d24d0fef1434fbd14567435a363d9",
+]
```

Comment:
@jtakalai did you mean to check in Infura URLs? Are these public?

Reply:
Yea but restricted to DATA and Marketplace contract. Feel free to delete. It was for testing the different networks with dev docker setup, but maybe not necessary for tests or anything.